source/stylesheets/_base.css.scss

```diff
@@ -11,6 +11,12 @@ header {
   margin: 0px auto;
   max-width: 960px;
   text-align: center;
+  
+  img {
+    width: 25%;
```

Comment:
I kinda dug how huge the image was on mobile, being full-width there. Was totally nutty on non-mobile, tho.